cmd/zdb/zdb.c

```diff
@@ -3594,11 +3596,12 @@ dump_l2arc_log_blocks(int fd, l2arc_dev_hdr_phys_t l2dhdr)
 		case ZIO_COMPRESS_OFF:
 			break;
 		case ZIO_COMPRESS_LZ4:
+		case ZIO_COMPRESS_ZSTD:
```

Comment:
I took a brief look at the persistent L2ARC changes in this PR. Looks good, but why is this necessary?

Reply:
if you take a look into my tree you wont find this patch. this is not neccessary and patch i merged from you. so the default variant you used is correct .we already discussed it if you remember